hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmBucketInfo.java

```diff
@@ -212,6 +266,8 @@ public static Builder newBuilder() {
     auditMap.put(OzoneConsts.STORAGE_TYPE,
         (this.storageType != null) ? this.storageType.name() : null);
     auditMap.put(OzoneConsts.CREATION_TIME, String.valueOf(this.creationTime));
+    auditMap.put(OzoneConsts.OBJECT_ID, String.valueOf(this.getObjectID()));
```

Comment:
I don't think we need to add these things to audit logs, as these are internal info to OM.
As in the audit log, if we have info of user passed information, we should be good.

Reply:
I added as we have this information in Audit Log for VolumeInfo. Do you think this information will not be useful for debugging lets say?

Reply:
Yaa agreed, but recently in one of PR a discussion came up that audit log is for to find information who did what operation, but not for debugging. So, if needed during debugging, we can log this information during performing the actual operation. 

Having precise/required information in audit logs will help when flushing audit logs and also the size of the audit log. So, I feel having this information is not really required. Thoughts?

Reply:
Sure. I will remove this info from audit log.